src/main/java/com/iexec/common/replicate/ReplicateStatus.java

```diff
@@ -20,6 +20,9 @@
     COMPUTED,
     COMPUTE_FAILED,
     CAN_CONTRIBUTE,
+    CANT_CONTRIBUTE_SINCE_DETERMINISM_HASH_NOT_FOUND,
```

Comment:
We should update getUncompletableStatuses() and co

Reply:
Done, thank you!